### ISSUE TITLE
SA1009 must not be triggered when the line is empty

### ISSUE THREAD

**Issue by xperiandri** [DotNetAnalyzers/StyleCopAnalyzers#2155]
Because of SA1009 I can't write

``` C#
                    new SoundPoint
                    (
                        resultWork.SoundPowerOut62,
                        resultWork.SoundPowerOut125,
                        resultWork.SoundPowerOut250,
                        resultWork.SoundPowerOut500,
                        resultWork.SoundPowerOut1000,
                        resultWork.SoundPowerOut2000,
                        resultWork.SoundPowerOut4000,
                        resultWork.SoundPowerOut8000
                    ), // error here
```

SA1009 must not be triggered if the line contains only spaces.

**Comment by sharwell** [comment]
:memo: This appears to be related to (but not a duplicate of) #1206.

**Comment by bjornhellander** [comment]
@sharwell, I was under the impression that this was intended behaviour. This code currently violates three rules and the documentation for SA1009 explicitly forbids this case. Does this really need discussion or can it be closed?

**Comment by vweijsters** [comment]
Seems like intended behavior to me as well and I see no reason to change it.

**Comment by xperiandri** [comment]
The rule is bad for pretty readable formatting.
It can be split this rule into two rules. If someone would like to exclude my case he will enable second rule.

**Comment by sharwell** [comment]
Took me a bit to realize exactly what was going on here. Yes, both the `(` and `)` have been misplaced in the example (in each case they should be at the end of the previous line). The requested situation may be allowable/configurable for the ongoing work to bring code style checking to Roslyn, but for StyleCop Analyzers this is a pretty clear case of two things:

1. (Objective) The rule is currently behaving as-originally-intended for this case.
2. (Subjective) The particular selection of rules related to the positioning of parentheses and commas for StyleCop produces the most readable overall layout for nested calls and long argument lists I have ever seen, with no alternative even coming close.

The former shows that this is not a bug, and the latter shows that there is no pressing need to support an exception in this scenario.
